examples/exec-healthz/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM busybox
```

Comment:
busybox is as lightweight as it gets

On Wed, Jul 8, 2015 at 11:13 AM, Prashanth B <notifications@github.com>
wrote:

> In examples/exec-healthz/Dockerfile
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10925#discussion_r34179830>
> :
>
> > @@ -0,0 +1,20 @@
> > +# Copyright 2015 The Kubernetes Authors. All rights reserved.
> > +#
> > +# Licensed under the Apache License, Version 2.0 (the "License");
> > +# you may not use this file except in compliance with the License.
> > +# You may obtain a copy of the License at
> > +#
> > +#     http://www.apache.org/licenses/LICENSE-2.0
> > +#
> > +# Unless required by applicable law or agreed to in writing, software
> > +# distributed under the License is distributed on an "AS IS" BASIS,
> > +# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +# See the License for the specific language governing permissions and
> > +# limitations under the License.
> > +
> > +FROM busybox
>
> I would like a shell. Can look for a lighter base if you prefer.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10925/files#r34179830>
> .
>